sklearn/linear_model/tests/test_ridge.py

```diff
@@ -48,7 +48,6 @@
     ignore_warnings,
 )
 from sklearn.utils.fixes import (
-    _IS_32BIT,
```

Comment:
I wonder why after my last push, this recent addition isn't imported from fixes anymore... Ruff moved it up.

Reply:
Okay, that not surprisingly raised. I moved it back to where it was and ruff seems to be content with it.